### ISSUE TITLE
Automatically create bluetooth interfaces

### ISSUE THREAD

**Issue by markfeathers** [embeddedTS/distro-seed#26]
https://unix.stackexchange.com/a/310918

In the event wlan0 is detected for silex/ti/microchip, we should have udev start a systemd service that calls btattach with the appropriate arguments for each chipset. We should aim to make this discover as much as it can from the device tree.

**Comment by ts-kris** [comment]
I agree with this. I was planning on exploring the kernel/DTS interface a bit more once I get deeper in the TS-7680 port, but if you have the time first then great.
